operator/deploy/deploy-kiali-operator.sh

```diff
@@ -568,7 +568,7 @@ delete_operator_resources() {
     ${CLIENT_EXE} delete --ignore-not-found=true customresourcedefinitions --selector="app=kiali-operator"
 
     # now purge all operator resources
-    ${CLIENT_EXE} delete --ignore-not-found=true all,sa,deployments,roles,rolebindings,clusterroles,clusterrolebindings --selector="app=kiali-operator" -n "${OPERATOR_NAMESPACE}"
+    ${CLIENT_EXE} delete --ignore-not-found=true all,sa,deployments,roles,rolebindings,clusterroles,clusterrolebindings,podsecuritypolicies --selector="app=kiali-operator" -n "${OPERATOR_NAMESPACE}"
```

Comment:
This same kind of change should be applied to the Makefiles where appropriate. I just looked and I think there is only one place where you need to change - it is here: https://github.com/kiali/kiali/blob/v1.2/operator/Makefile#L155

Make sure that Makefile is deleting the same resources as this line 571 of deploy-kiali-operator.sh is